uh50_api/uh50_reader.py

```diff
@@ -0,0 +1,69 @@
+""" 
+Reads raw UH50 response data from the UH50 unit. 
+To test the connection use validate, which will return the model name.
+"""
+import serial
+from serial import Serial
+
+
+class UH50Reader:
+    def __init__(self, port) -> None:
+        self._port = port
+        self.model: str

```

Comment:
variabele declaraties zijn niet nodig in python maar het mag wel. Als je ze wel wilt hebben, dan horen ze op class niveau.

Reply:
mja ik dacht voor de typehint. Maar niet zo boeiend, ik haal 'm weg.